pylibczi/tests/test_czi_file.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+from pylibczi import CziFile
+from pathlib import Path
+
+
+@pytest.mark.parametrize("fname", [
+    pytest.param('', marks=pytest.mark.raises(exception=IsADirectoryError))
+])
+def test_is_a_directory(data_dir, fname):
+    czi = CziFile(str(data_dir))
```

Comment:
It is already a `string`. Did you mean to test with both `pathlib.Path` and `str` here?